### ISSUE TITLE
WebXR polyfill support for iPhone

### ISSUE THREAD

**Issue by danrossi** [mrdoob/three.js#26624]
### Description

It seems IOS 17 is not getting WebXR support as expected and according to this comment it's going to only be available in VisionOS only. Which is impossible to develop for unless own mac hardware as they broke virtual machine support and the latest SDK requires the latest macOS which doesn't work in virtual machines anymore. 

https://bugs.webkit.org/show_bug.cgi?id=208988#c28

iPhone has been missing a WebXR polyfill support since end of 2019 in three.js but the polyfill project works in Babylon.JS without issues. 

I tried looked into a lens distortion effect for post processing which I had to migrate and build but I couldnt get it to render right with WebXR and process each eye seperately as none of it is supported yet. 

People keep asking for it at least for confirmation. iPhone is basically still legacy at this stage. 

### Solution

Either get the WebXR polyfill working again with three.js with equi or stereo video or using lens distortion with a stereoeffect as a work around. . 

https://github.com/pmndrs/postprocessing/blob/dev/src/effects/LensDistortionEffect.js

### Alternatives

none

**Comment by Mugen87** [comment]
This issue has already been discussed in #26166. 

Sorry but it is not the responsibility of `three.js` to make the WebXR polyfill work with the engine again. We are not in favor of cardboard workflows anyway, see https://github.com/mrdoob/three.js/issues/26166#issuecomment-1569435205.

> or using lens distortion with a stereoeffect as a work around.

Then let's focus on #8146.

**Comment by danrossi** [comment]
I closed the other one already all good, but now I managed to glean what is going on in webkit and what is being supported . I will wait for those WebXR fixes then keep trying to get the lens distortion integration I added into postprocessing working for both eyes. I had worked on changes in the fork but its applied fullscreen in the end to half each eye not to each eye and is overlapped. I'll keep trying to find something. 

I know it's silly and time wasting and Iphone and IOS is basically legacy as far XR goes with no chance of getting it. But I guess people still want some kind of stereo feature for confirmation. So maybe a stereo effect with the lens distortion and orientation controls. I might try that route while the PR is being worked on.
